lib/registries/addon/application/route.ts

```diff
@@ -0,0 +1,24 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import DS from 'ember-data';
+
+import Analytics from 'ember-osf-web/services/analytics';
+import BrandService from 'registries/services/brand';
+
+import BrandModel from 'ember-osf-web/models/brand';
+
+import RegistrationProvider from 'ember-osf-web/models/registration-provider';
+
+export default class ApplicationRoute extends Route {
+    @service analytics!: Analytics;
+    @service store!: DS.Store;
+    @service brand!: BrandService;
+
+    model() {
+        return this.store.findRecord('registration-provider', 'osf', { include: 'brand' });
```

Comment:
Right now the provider id is hardcoded to `osf`. Where should this come from?

Reply:
how about let's put it in the config (somewhere [under `OSF`](https://github.com/robustdj/ember-osf-web/blob/19cc7fcc51a444c0cebab13475bcc2cd0f833a6d/config/environment.js#L146))? we might make it more dynamic in the future but for the time being, it'll always be `'osf'`